index.test.js

```diff
@@ -1,9 +1,21 @@
-'use strict'
-
-const { test } = require('tap')
-const sinon = require('sinon')
-
-const setup = require('./test-setup')
+import { test } from 'node:test';
+import assert from 'node:assert/strict';
+import sinon from 'sinon';
+
+import setup from './test-setup.js';
+
+test('Ensure all imports used by index.js are correct', async t => {
+  try {
+    await import('./lib/github.js')
+    await import('./lib/auth.js')
+    await import('./lib/routes.js')
+    
+    assert.ok(true, 'all imports used by index.js import correctly')
```

Comment:
not mandatory, but you can use `t.assert` everywhere

Reply:
Happy with it either way, if you'd prefer I update it just thumbs up this comment and I'll push up the change.

Reply:
no need, more of a FYI